packages/experimental/Badge/src/PresenceBadge/PresenceBadge.types.ts

```diff
@@ -1,11 +1,13 @@
 import { BadgeProps } from '../';
+import { StyleSheetProperties } from 'react-native';
 
 export const presenceBadgeName = 'PresenceBadge';
 export type Presence = 'doNotDisturb' | 'busy' | 'unknown' | 'blocked' | 'outOfOffice' | 'away' | 'available' | 'offline';
 
 export interface PresenceBadgeProps extends BadgeProps {
   presence?: Presence;
   isOutOfOffice?: boolean;
+  styles?: StyleSheetProperties;
```

Comment:
Is this styles prop supposed to be like styles on the View component? The `style` prop on the View component is type `StyleProp<ViewStyle>`. Also, for consistency with other components, I think it should be named `style`, without the `s`

Reply:
I didn't use it as a style because it's part of styles for object which is passed to customize method in PresenceBadge component. It's sad that it's not possible to do "double customize" :) PrecenseBadge uses Badge.customize and I still need to customize PresenceBadge. I'll try to find a better solution for passing styles

Reply:
I did the same thing as we did with different kinds of Buttons. I think currently it's the best solution

Reply:
Can you explain what you mean by not being able to double customize? You can call customize/compose on a customized component, maybe I'm misunderstanding what you mean?

Reply:
Yes, you understand correctly. I wanted to reuse existing component and not to have the same parts of the code. But this solution didn't work so I used the same approach we have for the Button component.

Reply:
Hmm I'm curious about looking into this for learning/understanding but changing the PresenceBadge to not use customize seems like a reasonable way forward for the PR.